db/migrate/20150611171433_add_admin_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean
+    User.all.each do |user|
```

Comment:
This is not a good pattern. You should not use data-migrations with schema migrations. This may lead to bugs that are hard to track in migrations. You can not revert back this right now.

To propose a better way: you should build a custom rake task. Check this guide - http://guides.rubyonrails.org/command_line.html#custom-rake-tasks